### ISSUE TITLE
[Bug] Not spawning endermen in the end while I am elsewhere

### ISSUE THREAD

**Issue by djcprone** [SuperMartijn642/ScarecrowsTerritory#29]
**Version Info**
- Minecraft 1.20.1, forge 47.1.3
- Modpack TNP Limitless 7 v0.28
- Scarecrow's Territory, 1.1.11b-forge-mc1.20

**Description of the Bug**

I have a system set up in the end, big ol chunkloaded (via ftb) platform with vector plates to bring the naturally spawned (as in, no spawner) endermen into range of the mob grinders. This works when I am standing there within 24 blocks but not when I /home to the overworld or elsewhere. Can be verified because I can watch the mob loot stop increasing in my system the second I go elsewhere, can watch it climb rapidly while there, and see the endermen. So, I got this mod thinking that would do the trick. I also increased the range of passive spawning in the config to 24.0 to test if it was a range thing. No luck.

**Steps to Reproduce**

I guess replicate above?

**Screenshots**

I don't think a screenshot can really demonstrate so I did a recording and uploaded to vimeo. You can see it is 1. chunkloaded, 2. works when I am there, and 3. does not work when I am not.

https://vimeo.com/881157165?share=copy

In essence, it pretty much works the same as it did before I ever added the mod or the block to the build.

**Comment by SuperMartijn642** [comment]
So if I am understand the farm right, it just moves the enderman to the center where they are killed by grinders right? Mobs may despawn as soon as they leave the range of the scarecrow. If that is indeed how it works, that should be fine.

Also, could you share what other mods you have installed?

**Comment by djcprone** [comment]
> So if I am understand the farm right, it just moves the enderman to the center where they are killed by grinders right? Mobs may despawn as soon as they leave the range of the scarecrow. If that is indeed how it works, that should be fine.
> 
> Also, could you share what other mods you have installed?

That is correct. Without your mod, if I just hover there or even just stand up on that platform, it is close enough range that they'll just keep spawning like crazy and the farm works as intended. But without a player nearby, nothing spawns and it stops working. I was thinking your mod should solve that problem. However, results are the same even with the scarecrow.

Regarding despawning, since the plates move the mobs in toward the scarecrow and the range for passive is set to 24 + range of 10 for despawning in the config, there shouldn't be an issue there, right? 

The mod list is at the bottom of this page: https://legacy.curseforge.com/minecraft/modpacks/tnp-limitless-7/files/4819624

**Comment by SuperMartijn642** [comment]
> there shouldn't be an issue there, right?

Yeah indeed, then it should keep them spawning, so seems like a bug.

> The mod list is at the bottom of this page: https://legacy.curseforge.com/minecraft/modpacks/tnp-limitless-7/files/4819624

Thank you, I also realize now that you did already mention the modpack in your original post, whoops 😅

I am somewhat busy with exams, so might be a bit before I get to testing this

**Comment by djcprone** [comment]
> Yeah indeed, then it should keep them spawning, so seems like a bug.

Alright, good to know I wasn't messing up at least, haha.

> I am somewhat busy with exams, so might be a bit before I get to testing this

No worries, I appreciate your work. Good luck with exams and just reply here when you get to it so I see it in my email please!

**Comment by djcprone** [comment]
> Yeah indeed, then it should keep them spawning, so seems like a bug.

I wanted to add, if it helps debug, that it seems to work as long as I am in the dimension. So if I am in the end with the scarecrow, all is well. If I go to another dimension, it stops. It does work if I'm still in the end though, which I hadn't realized prior. My platform above is outside of vanilla spawn range so I realized after I removed the mod and they stopped spawning when I was up there. (They were with your scarecrow). So I put it back to confirm. Hopefully that makes it easier for you to weed out the problem!

**Comment by Naireem-sudo** [comment]
Yeah, same problem here... Any news on this?

**Comment by Jonaykon** [comment]
EXACT same issue for me

**Comment by Jonaykon** [comment]
found this in the changelog for version 1.1.10 "Allow mobs spawned by a scarecrow to despawn when there's no player in the same dimension", i think that could be the cause

**Comment by Jonaykon** [comment]
downgrading to version 1.9 of the mod fixes the problem!
